src/System.Private.Xml/tests/Xslt/XslTransformApi/CXslTransform.cs

```diff
@@ -523,13 +506,18 @@ public void XmlResolver7(InputType inputType, ReaderType readerType, TransformTy
             {
                 _output.WriteLine(e.ToString());
                 _output.WriteLine("Could not copy file to local. Some other issues prevented this test from running");
-                return; //return TEST_SKIPPED;
+                throw;
             }
 
+            const string expected = @"<?xml version=""1.0"" encoding=""utf-8""?><result>123</result>";
             if (LoadXSL("xmlResolver_document_function_absolute_uri.xsl", inputType, readerType) == 1)
             {
-                if ((Transform("fruits.xml", transformType, docType) == 1) && (CheckResult(377.8217373898, transformType) == 1))
+                xslt.XmlResolver = new XmlUrlResolver();
+                if (Transform("fruits.xml", transformType, docType) == 1)
```

Comment:
Similar Assert.Equal comment with regards to being able to remove the below Assert.True(false).  Will stop commenting on these, too.